functions.j2

```diff
@@ -187,13 +72,27 @@ function {{ function }}() {
   done
 {%- endif %}
 
+  easy_infra_{{ function | replace("-", "_") }}_hooks=()
+  if [[ "${DISABLE_HOOKS:-false}" != "true" ]]; then
+    # Dynamically register hooks
+    for file in /opt/hooks/bin/*.sh; do
+      command=$(awk -F\: '/register_hook/ { gsub(/ /,""); print $2 }' "${file}")
```

Comment:
This means that if a command has multiple aliases (like `ansible` and `ansible-playbook`), it will need to be registered against both.  I will add this to the docs as an important note